webapp/src/components/incoming_calls/call_incoming_condensed.tsx

```diff
@@ -27,23 +27,36 @@ type Props = {
 
 export const CallIncomingCondensed = ({call, onWidget = false, joinButtonBorder = false, className}: Props) => {
     const {formatMessage} = useIntl();
-    const teammateNameDisplay = useSelector(getTeammateNameDisplaySetting);
+    const messageRef = useRef<HTMLDivElement>(null);
     const caller = useSelector((state: GlobalState) => getUser(state, call.callerID));
+    const [callerName, others] = useGetCallerNameAndOthers(call, 2);
+    const [showTooltip, setShowTooltip] = useState(false);
 
     useRingingAndNotification(call, onWidget);
     const [onDismiss, onJoin] = useDismissJoin(call.channelID, call.callID);
 
-    const callerName = displayUsername(caller, teammateNameDisplay, false);
+    useLayoutEffect(() => {
```

Comment:
What's the reasoning behind using `useLayoutEffect` vs `useEffect` in this case?

Reply:
I defaulted to that because it's a layout calculation, but you're right -- there's no chance that we'll see a rerender, so it's fine to use useEffect.